app/system/language/en/lang.php

```diff
@@ -471,7 +471,7 @@
         'help_approve_reviews' => 'Approve new review entry automatically or manually',
         'help_default_location' => 'Choose or add a new location to set as your main/default restaurant location.',
         'help_default_order_status' => 'Select the default order status when a new order is placed/received',
-        'help_processing_order_status' => 'Select the order status an order must reach before the order starts stock reduction and coupon redemption',
+        'help_processing_order_status' => 'Select the order status an order must reach before the order starts stock reduction',
```

Comment:
my guess is coupon redemption should do done once payment is confirmed?

Reply:
Yeah I just thought it best to update the text as there are no coupons in core?

Reply:
Cool